### ISSUE TITLE
In 0.6.0 (store), pass can only decrypt a single record, fails for subsequent decryptions, requires app restart

### ISSUE THREAD

**Issue by opposablebrain** [mssun/passforios#286]
TL;DR: if you have decryption issues, reimport gpg key
----------------------------------------------------------
To reproduce (this is the same whether or not I choose to save gpg passphrase):
1. Launch pass
2. Tap on any record
3. App prompts for gpg passphrase
4. Enter gpg passphrase
5. Record is correctly decrypted and displayed
6. Go back to main list and tap on any record (can be the same as in step 2)
7. App displays error "Cannot show password // Cannot decrypt password [Cancel | Try Again]"
8. [Try Again]
9. App prompts for gpg passphrase
10. Enter gpg passhprase
11. App displays error "Cannot show password // Cannot decrypt password [Cancel | Try Again]"
12. Hard-exit the app
13. Back to step 1

This is fixed by removing and reimporting my gpg key (4096 rsa)

**Comment by yishilin14** [comment]
Thank you for your detailed report. Someone else also reported a similar problem via email yesterday. I will take a look this weekend.

**Comment by urbanslug** [comment]
I'm experiencing something close to this in version 0.7.0 and whille using subkeys.
It accepts my passphrase when I import the keys during the proompt to remember the passphrase.

Mine is more severe in that I can't get it to decrypt even after a phone restart.

**Comment by urbanslug** [comment]
Update, it worked, at least in 0.7.0(15). It works just fine. The problem may have been caused by my exporting the subkeys with `--export-secret-keys` instead of ` --export-secret-subkeys`.
